data/provider/src/main/java/tech/pegasys/teku/api/NodeDataProvider.java

```diff
@@ -178,37 +178,40 @@ private SafeFuture<Optional<SubmitDataError>> addBlsToExecutionChange(
             });
   }
 
-  public void subscribeToReceivedBlocks(ImportedBlockListener listener) {
+  public void subscribeToReceivedBlocks(final ImportedBlockListener listener) {
```

Comment:
unrelated things like this we can do as trivial PRs and not pollute PRs... keep that in mind next time? makes easier to review.